flowetl/Dockerfile

```diff
@@ -32,6 +38,11 @@ RUN pip install /opt/etl
 
 USER airflow
 COPY --chown=airflow:airflow ./dags/* ${AIRFLOW_HOME}/src/
+
+# Needed on $AIRFLOW_HOME so that different users passed through --user
+# have the permission to create and use their files for Airflow
+RUN chmod -R 777 ${AIRFLOW_HOME}
```

Comment:
PG image does this, and then replaces with 700 when the container actually runs. Any specific reason not to do similarly?

Reply:
In PG's case, it does the chmod when run as root, and `chmod 700 "$PGDATA" 2>/dev/null || :` which allows it to fail if chown also fails. In our case, if it's not started with uid 0, the chown and chmod would fail because that uid/gid won't be the owner or root, but I agree that's a good idea to do it _when possible_. Added that.